### DISCUSSION TITLE
Can I publish documents publicly without the need for documentation and password?

### DISCUSSION THREAD

**Discussion by RajveerRall** [docmost/docmost#123] [Help / Questions]
Hey Docmost team, 

I already love this project. However, can I create a public documentation using it? If not, this would be a great feature!

Thanks

**Comment by AMDphreak** [comment]
It's pretty sad they don't have this functionality. How is anyone supposed to make a community wiki without a public wiki?

**Reply by AMDphreak** [reply]
The maintainer responded saying it is a planned feature. https://github.com/docmost/docmost/discussions/82#discussioncomment-10056020
